### ISSUE TITLE
Mesure TeamMail data // batterie usage

### ISSUE THREAD

**Issue by chibenwa** [linagora/tmail-flutter#1656]
### Description

In order to promote the usage of Team Mail, we need to evaluate TeamMail impact on the end user device.

We wishes to confirm that TeamMail consume less data and less batterie than a classical mail application.

Now, let's prove it!

### Methodology

 -> Use 3G / 4G in order to track per app data usage
 -> Use a fully charged rebooted phone to track per app batterie usage

Then run the **scenario** on both apps, and depending of the results, modify the README with the claims.

**Scenario**: 

```
 - Switch mailbox 100 times
 - Do 100 manual refresh
 - Using thunderbird send 25 mails to the email account and wait notifications with the application in the background
 - Send 25 emails to yourself, 
 - Move these 50 emails to a folder (one by one)
```

Applications:
  - TeamMail latest version
  - Samsung Email applicationm configured with IMAP
  
@QuangHoang1210 that task is for you ;-)

### Screenshots

![Screenshot_20230325-112314_Device care](https://user-images.githubusercontent.com/6928740/227714019-226eeb25-0d41-4452-883a-497746d17ed1.jpg)

![Screenshot_20230325-114350_Settings](https://user-images.githubusercontent.com/6928740/227714020-29534484-4773-4c65-affa-bb15495fcdb1.jpg)

**Comment by QuangHoang1210** [comment]
Report link: https://docs.google.com/spreadsheets/d/1tYxjB4tdbtz4KBZlttaK_VsjTcT2pOxf-2Pj2JXiP7w/edit?usp=sharing

Because of Thunderbird blocking issue. Then I will try to finish the last round testing today <Thunderbird sending email>
@chibenwa  @hoangdat  @dab246

**Comment by chibenwa** [comment]
Looks nice.

Can we measure that for k9mail too?

Also can you add "open 50 emails"?

I also suqtect you email account to be ammost empty, quang... Thus it might not be "that representative".

**Comment by QuangHoang1210** [comment]
@chibenwa  yup. I fake more than 150 email on mailbox. Is that make sense!? Btw I also config to the preprod account to check it again.

K9mail -- yes, I will.
Open 50 emails -- Yes, I will add this scenario.

**Comment by chibenwa** [comment]
150 is a bit short. 5000 - 10000 is a bit more reasonable...

**Comment by Arsnael** [comment]
The accounts on preprod logically have a big amount of mails no?
